src/scaffold/emitter.rs

```diff
@@ -236,6 +238,23 @@ impl<'s> Visitor for EmitterI<'s> {
 
         Ok(emitted)
     }
+
+    fn visit_statement(
```

Comment:
good bot. These are actually never reached (tried adding a test here, it's only the fmt which is emitting the statements, which kinda look weird actually, or I misunderstood how the emitter/formatter are working together)
 
It's the last pending thing iic, rest has been fixed

Reply:
Note that `emitter` is used by `bulloak scaffold` and the formatter is used by `bulloak check`. Though I'm not sure which formatter you are referring to.

Reply:
Yeah, wasn't super clear, my bad.

Latest commit fixed what I meant (the emitter wasn't emitting the function call)